apps/communications/dialer/test/marionette/lib/views/activities/views.js

```diff
@@ -10,15 +10,24 @@ function ActivitiesView(client) {
 
 ActivitiesView.prototype = {
   pickCreateNew: function() {
-      this.client.waitFor(function() {
-        return this.accessors.addNewContactMenuItem.enabled();
-      }.bind(this));
-      var button = this.accessors.addNewContactMenuItem;
-      button.tap();
-
+      this._openElement(this.accessors.addNewContactMenuItem);
+      
       var contacts = this.client.loader.getAppClass('contacts');
       contacts.switchToCreateNewContactActivity();
       return contacts;
+  },
+
+  pickAddToContact: function(){
+      this._openElement(this.accessors.addToContactMenuItem);
+
+      var contacts = this.client.loader.getAppClass('contacts');
+      contacts.switchToApp();
+      return contacts;
+  },
+
+  _openElement: function(accessorToTap){
+      var button = accessorToTap;
+      button.tap();
```

Comment:
Please add the returns there. This will help maintaining the test. Let's say 1 day we changed the way to get the class of the contact app, we will only require to change this function, instead of many ones.